.github/workflows/config/spelling_allowlist.txt

```diff
@@ -28,6 +28,7 @@ DOI
 Deuteron
 Doxygen
 FPGAs
+Fermioniq
```

Comment:
Nit: if `fermioniq` is an allowed word, then we don't need to explicitly add the capitalized version of this word, so I would recommend removing this line.
```suggestion
```

Reply:
Ok that is fine with me.